classes/webservice/WebserviceSpecificManagementImages.php

```diff
@@ -1221,6 +1221,7 @@ protected function writePostedImageOnDisk($reception_path, $dest_width = null, $
                     return true;
                 }
             }
+            throw new WebserviceException('Missing tmp_file property. Check supplied input data.', [67, 400]);
```

Comment:
@KminekMatej I think you'd rather reuse what had been done here : https://github.com/PrestaShop/PrestaShop/blob/develop/classes/webservice/WebserviceSpecificManagementImages.php#L1111-L1113

Reply:
You, sir, are absolutely right. I will update that